src/pages/_id/index.vue

```diff
@@ -221,44 +270,34 @@ export default {
       }
     },
   },
-  async asyncData({ route, $api, error, store }) {
+  async asyncData({ route, $api, error, store, redirect }) {
     let { id } = route.params;
     if (id && isValidAddress(id)) {
       if (id.startsWith('cosmos1')) {
         id = convertAddressPrefix(id, 'like');
       }
       if (id.startsWith('like1')) {
         try {
-          const userInfo = await store.dispatch('fetchUserInfoByAddress', id);
-          return {
-            userInfo,
-            wallet: id,
-          };
+          await store.dispatch('fetchUserInfoByAddress', id);
+          return;
         } catch (error) {
           // eslint-disable-next-line no-console
           console.error(error);
         }
-        return {
-          wallet: id,
-          userInfo: undefined,
-        };
```

Comment:
Why is this removed? like1 without likerId should be valid user, or another `return` is needed here?